### ISSUE TITLE
Use flow-typed to install third-party type declarations

### ISSUE THREAD

**Issue by zacharygolba** [george-haddad/tool-belt#17]
[flow-typed](flowtype/flow-typed) is a must have for projects that use [flow](https://flow.org). It allows you to install community maintained declarations for packages you depend on based on your `package.json` file.

To get started, run the following commands:

```sh
cd tool-belt
yarn add --dev flow-typed
yarn flow-typed install
```

Now that the declarations are installed to `flow-typed`, you can run the type checker and then commit the declaration files.

```sh
yarn flow check
git commit -am "chore(types): add flow-typed declarations"
git push
```

**Comment by george-haddad** [comment]
Thanks! would it make sense to add `flow check` as part of my tests

```
"test": "eslint src && flow && flow check"
```

**Comment by zacharygolba** [comment]
No problem! I think that's a great idea. Although you can probably just run `flow check` or `flow` as `flow && flow check` would be redundant. If I remember correctly `flow` uses cache and is incremental where `flow check` is not. I may be wrong though.

**Comment by george-haddad** [comment]
Yup, pretty much the same thing according to https://stackoverflow.com/questions/38902752/whats-the-difference-between-running-flow-and-flow-check?rq=1
